pkg/cfaws/ssotoken.go

```diff
@@ -32,12 +32,22 @@ type SSOPlainTextOut struct {
 // we'll allow users to export a plaintext token from their keychain for compatibility
 // purposes with other AWS tools.
 func CreatePlainTextSSO(awsConfig config.SharedConfig, token *securestorage.SSOToken) *SSOPlainTextOut {
+	ssoRegion := awsConfig.SSORegion
+	if ssoRegion == "" && awsConfig.SSOSession != nil {
```

Comment:
Was originally

```
if awsConfig.SSOSession != nil {
  ssoRegion = awsConfig.SSOSession.SSORegion
  ssoStartURL = awsConfig.SSOSession.SSOStartURL
}
```

Changed as profile can still have URLs set and the config still uses that